Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_test.py

```diff
@@ -2924,3 +2938,110 @@ def test_generate_login_url(mocker):
                    f'&client_id={client_id}&redirect_uri={redirect_uri})'
     res = MicrosoftDefenderAdvancedThreatProtection.return_results.call_args[0][0].readable_output
     assert expected_url in res
+
+
+def test_get_file_statistics_command(mocker):
+    """
+    Given:
+    - SHA1 File hash
+
+    When:
+    - Calling the get_file_statistics_command function
+
+    Then:
+    - Assert correct context output and raw response
+    """
+    from MicrosoftDefenderAdvancedThreatProtection import get_file_statistics_command
+
+    # Set
+    mocker.patch.object(client_mocker, 'get_file_statistics', return_value=FILE_STATISTICS_API_RESPONSE)
+    context_to_response_key_mapping: dict = {
+        'GlobalPrevalence': 'globalPrevalence',
+        'OrgPrevalence': 'orgPrevalence',
+        'GloballyPrevalence': 'globallyPrevalence',
+        'OrganizationPrevalence': 'organizationPrevalence',
+        'OrgFirstSeen': 'orgFirstSeen',
+        'OrgLastSeen': 'orgLastSeen',
+        'GlobalFirstObserved': 'globalFirstObserved',
+        'GlobalLastObserved': 'globalLastObserved',
+        'TopFileNames': 'topFileNames',
+    }
+
+    # Arrange
+    results = get_file_statistics_command(client_mocker, {'file_hash': '0991a395da64e1c5fbe8732ed11e6be064081d9f'})
+    statistics_context_output = results.outputs['Statistics']
+
+    # Assert
+    for context_key, response_key in context_to_response_key_mapping.items():
+        assert statistics_context_output[context_key] == FILE_STATISTICS_API_RESPONSE[response_key]
+
+    assert results.raw_response == FILE_STATISTICS_API_RESPONSE
+
+
+def test_get_file_statistics_indicator():
+    """
+    Given:
+    - SHA1 File hash and Defender for Endpoint file statistics API response
+
+    When:
+    - Calling the get_file_statistics_indicator function
+
+    Then:
+    - Assert correct indicator field values
+    """
+    from MicrosoftDefenderAdvancedThreatProtection import get_file_statistics_indicator
+
+    # Set
+    indicator_to_response_key_mapping: dict = {
+        'GlobalPrevalence': 'globallyPrevalence',
+        'OrganizationPrevalence': 'organizationPrevalence',
+        'OrganizationFirstSeen': 'orgFirstSeen',
+        'OrganizationLastSeen': 'orgLastSeen',
+        'FirstSeenBySource': 'globalFirstObserved',
+        'LastSeenBySource': 'globalLastObserved',
+    }
+
+    # Arrange
+    file_indicator = get_file_statistics_indicator('0991a395da64e1c5fbe8732ed11e6be064081d9f', FILE_STATISTICS_API_RESPONSE)
+    indicator_data: dict = next(iter(file_indicator.to_context().values()))
+
+    # Assert
+    for indicator_key, response_key in indicator_to_response_key_mapping.items():
```

Comment:
I know we discussed this and said we could either do a loop or compare dictionaries and it doesn't matter, now when I think about it again, looping misses the case where the one we don't loop over has a different number. Prefer `assert dict1 == dict2`, pytest will tell us what's wrong.

Reply:
Fixed in the updated unit tests in commit e9f66b0abb115a5d3d2067356411fb235763b986